tests/PortingAssistant.Client.UnitTests/PortingAssistantNugetHandlerTest.cs

```diff
@@ -306,7 +306,7 @@ public void Setup()
             _fileSystem.Setup(fileSystem => fileSystem.FileExists(It.IsAny<string>())).Returns((string file) =>
             {
                 string solutionId;
-                using (var sha = new SHA256Managed())
+                using (var sha = SHA256.Create())
```

Comment:
did you check remaining parts of the code and also change there, seems like SHA256Managed is obselete

Reply:
Yes, I checked the rest of the code to replace the obsolete api. Only found 1 reference in src and 2 in the test directory